doc/classes/ProjectSettings.xml

```diff
@@ -426,6 +426,9 @@
 		<member name="debug/file_logging/max_log_files" type="int" setter="" getter="" default="5">
 			Specifies the maximum number of log files allowed (used for rotation).
 		</member>
+		<member name="debug/gdscript/warnings/enforce_static_typing" type="int" setter="" getter="" default="0">
+			If [code]enabled[/code] prints a warning or an error when a variable is not statically typed.
```

Comment:
This setting is not declared anywhere, `GLOBAL_GET` only reads the value

Reply:
Where do I declare the setting?

Reply:
My bad realised it is handled specifically, no need here I think

Reply:
On my own CI I got a "Process completed with exit code 1." error when building Linux with Mono on "Check for class reference updates." Does this occur every time a new property is added to a base class?

Reply:
The wording here should be more like that of other cases though, it's ignore/warn/error

Reply:
The warnings here have to be sorted alphabetically

Reply:
Marking as draft for now to not trigger the CI too much

Reply:
Oh you won't you're a first time contributor someone has to start it, and draft won't stop the CI, but good to keep in mind for the future

Reply:
Ok, good to know! Thanks

Reply:
If you look at the entry in the CI on your fork for that run it shows the documentation entry being removed and added in the proper location